sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -57,14 +57,19 @@ enum TraceLevel {
 // Return true if we want to trace PI related activities.
 bool trace(TraceLevel level);
 
+// Return PluginSpecificData referenced by kernel code
+__SYCL_EXPORT void *getPluginSpecificData(backend be);
```

Comment:
I don't understand this interface. How/where it will be implemented and used? Maybe just don't add it now in this PR.

Reply:
This new API is removed already.

Reply:
why do I still see this declaration in the changes then?

Reply:
It looks like I missed one. The function is removed in other places. Sorry for confusion.

Reply:
Removed.

Reply:
Thanks! So we ended up with no changes to PI API, which is a good thing.